cmd/bridge/main.go

```
@@ -240,6 +241,8 @@ func main() {
 		}
 	}
 
+	nodeArchitectures := serverutils.RemoveEmptyStrings(strings.Split(*fNodeArchitectures, ","))
```

@jhadvig Should we err if there are any empty strings (aside from the default `""`)? We might be masking a bug here if the operator ever passes one.

---

> We might be masking a bug here if the operator ever passes one.

when did that ever happen ? 🥇

---

will update ! :)

---

Updated with latest push.  @spadgett, PTAL.